static/package.json

```diff
@@ -58,9 +61,7 @@
     "typescript": "^4.7.4",
     "webpack": "^5.73.0",
     "webpack-cli": "^4.10.0",
-    "webpack-remove-empty-scripts": "^0.6.3",
-    "jszip": "^3.10.0",
-    "papaparse": "^5.3.2"
+    "webpack-remove-empty-scripts": "^0.6.3"
```

Comment:
please revert this change, jszip and papaparse are used in other components in the repo

Reply:
Thanks for the review! Sorry I think I blindly ran some npm commands without checking the aftereffects for deletions.  Thanks for catching!

Reply:
oh! I think jszip, papaparse are both still there (npm install might've sorted the list)